mk/deploy.mk

```diff
@@ -107,8 +111,17 @@ s3info%: guard-S3_VERSION_PATH .build-artefacts/requirements.timestamp
 s3activate := $(patsubst %,s3activate%,int,infra,prod)
 PHONY: $(s3activate)
 s3activate%: guard-DEPLOY_GIT_BRANCH \
+		         guard-SNAPSHOT \
+	guard-VERSION .build-artefacts/requirements.timestamp
+	${AWS_CMD} s3 cp $(S3_OPTS) --recursive s3://$(S3_BUCKET_$(shell echo $*| tr a-z A-Z))/${DEPLOY_GIT_BRANCH}/${SNAPSHOT}/  s3://$(S3_BUCKET_$(shell echo $*| tr a-z A-Z))/
+
+s3deploydist := $(patsubst %,s3activate%,dev,infra,prod)
+PHONY: $(s3deploydist)
+s3deploydist%: guard-DEPLOY_GIT_BRANCH \
+		         guard-SNAPSHOT \
+						 guard-S3_BUCKET_INT \
 	guard-VERSION .build-artefacts/requirements.timestamp
-	${PYTHON_CMD} ./scripts/s3manage.py activate --branch ${DEPLOY_GIT_BRANCH} --version ${VERSION} $(S3_BUCKET_$(shell echo $*| tr a-z A-Z));
+	${AWS_CMD} s3 cp $(S3_OPTS) --recursive s3://$(S3_BUCKET_INT)/${DEPLOY_GIT_BRANCH}/${SNAPSHOT}/  s3://$(S3_BUCKET_$(shell echo $*| tr a-z A-Z))/${DEPLOY_GIT_BRANCH}/${SNAPSHOT}/
 
```

Comment:
As clean as it would be, I don't think a simple copy will work:
for `master`:
- `robots_prod.txt` needs to be renamed to `robots.txt`
- the `cp` target is `/` and not `/<branch_name>/` as for a non-master branch

futhermore, the patsubst part has copy-paste error...